### ISSUE TITLE
[Accessibility]: Many selectable SVGs on Code monitoring getting started page

### ISSUE THREAD

**Issue by benvenker** [sourcegraph/sourcegraph#35156]
### Audit type

ARC Toolkit

### Problem description

Many/most of the SVGs on the getting started page are showing errors in ARC because they are selectable

### Expected behavior

I guess they shouldn't be selectable?

### Additional details

![image](https://user-images.githubusercontent.com/4605817/167487740-c2fb552a-6e03-4633-9878-15a84f063e69.png)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.
